### PR TITLE
Translation note for README.md

### PR DESCRIPTION
**Pull request by LiraLemur**
Signed-off-by: Sara <lira.lemur@gmail.com>

### Description of the Change

Added a note about translations being outdated

### Benefits

Less confusion by people seeing old docs

### Possible Drawbacks 

We'll need to remove it once the situation is resolved

### CODE REVIEW

**Comment by LiraLemur** [comment]
@MBoldyrev it looks like this:
image

**Comment by MBoldyrev** [comment]
this is limited to github. but users from the web come to readthedocs and see outdated docs

**Comment by LiraLemur** [comment]
@MBoldyrev  
> this is limited to github. but users from the web come to readthedocs and see outdated docs
That's being fixed by removing the translations from ReadTheDocs. I cannot fathom any other way of doing that except adding a note to every translation (which is also wrong as translation files should not be edited manually.

**Comment by MBoldyrev** [comment]
if you remove the translations, it is what I would do. I thought that this PR is all what is going to be done regarding the outdated docs.

File: README.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ We have documentation in several languages available and you are welcome to cont
...
 
 </center>
+
+> We are currently researching better ideas for translation platforms. That is why the documentation in outdated at the moment. Please, refer to the English documentation for now.
```
Comment (lebdron) [inline README.md]:
```suggestion
_We are currently researching better ideas for translation platforms. That is why the documentation in outdated at the moment. Please, refer to the English documentation for now._
```

**Pull request merged**